mantidproject/mantid
# 21187 add merge selection functionality to old sans gui

This PR adds the functionality whereby the merge region can be specified to the old SANS GUI. This should be merged after the PR which implements this on the backend and in the new GUI #21107. 

**To test:**
- Get the SANS data from http://download.mantidproject.org/. The ISIS sample data has some SANS data in a folder called loqdemo.
- Add this folder to your directories
- Open mantid, go to interfaces->SANS->ISIS SANS 
- Load the maskfile.txt from the folder as a user file.
- Load the batch_mode_reduction.csv as the batch file
- Go to the reduction tab and select merged reduction 
- Tick Use Range Box, put 0.1 and 0.2 in the two boxes to the left.
- Go to main tab and click 1D Reduce
- Plot all three output workspaces
- Confirm that the merged reduction matches LAB below 0.1, HAB above 0.2 and is an average of the two between these two values.

Code review


Fixes #21187 . 

**Release Notes** 
Release notes already added in #21107


---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
